dev-scripts/markdownDocFormatting.py

```diff
@@ -0,0 +1,25 @@
+import mmap, re, sys
+from glob import glob
+
+files_to_format = sys.argv[1:]
+print(f"files needed for markdown formatting: {files_to_format}")
+
+resource_docs_location = glob("./docs/resources/*", recursive=True)
+for file in files_to_format:
+  for mdFile in resource_docs_location:
+    if f"./docs/resources/{file}" == mdFile:
+      print(f"formatting {mdFile}...")
+      with open(rf'{mdFile}', 'r+') as file_to_edit:
+        file_content = file_to_edit.read()
+        file_content = re.sub('(## Schema\n\n```)','## Schema\n', file_content)
```

Comment:
If this is all this script is doing, can the template be updated to use something other than `codefile` so that it doesn't insert the backticks? Is just `file` something that exists and can be used?